src/KeyworkerManagement/App.js

```diff
@@ -179,21 +168,13 @@ class App extends React.Component {
       dispatchLoaded,
     } = this.props
 
-    const hasKWRoles = user && (user.hasKeyWorkerMonitor || user.writeAccess)
-    const hasAdminUtilities =
-      user && (user.maintainAccess || user.maintainAccessAdmin || user.maintainAuthUsers || user.groupManager)
-
     let innerContent
     const routes = (
       // eslint-disable-next-line
       <div className="inner-content" onClick={() => boundSetMenuOpen(false)}>
         <div className="pure-g">
           <Switch>
-            {hasAdminUtilities && <Route exact path="/" render={() => <Redirect to="/admin-utilities" />} />}
-            {hasKWRoles && <Route exact path="/" render={() => <Redirect to="/manage-key-workers" />} />}
-            {!(hasKWRoles || hasAdminUtilities) && (
```

Comment:
this doesn't look right - there is no condition on `/unauthorised` any more?

Reply:
That code was there to route you to the right page when landing on `/` but with only one of the authorities.  It didn't actually stop you from typing in `/manage-key-workers` to the address bar and so it wasn't really adding any security.